decidim-core/config/locales/en.yml

```diff
@@ -1072,7 +1076,6 @@ en:
           intro: 'There were the following errors with your message:'
           ok: OK
         index:
-          ago: ago
```

Comment:
I would also leave this in place, even if it's not necessarily used anywhere.

Reply:
This needs to be removed, otherwise the core spec related to unused strings will fail. The other way we could tackle this is to ignore it from i18n-tasks config.

Reply:
I would say let's add it to the ignores within this same PR to avoid the error.

I would prefer at least not to remove any strings in point releases.